Cards/MadGraph5_aMCatNLO/SingleT/TWminusto2L2Nu-5FS-DR1_TuneCP5_13p6TeV_amcatnlo-pythia8/TWminusto2L2Nu-5FS-DR1_TuneCP5_13p6TeV_amcatnlo-pythia8.json

```diff
@@ -0,0 +1,21 @@
+{
+    "template": "NLO_run_card.dat",
+    "template_user": [],
+    "template_vars":{
+        "parton_shower": "pythia8",
+        "ickkw": "0",
+        "maxjetflavor": "5",
+        "istr" : "1"
+    },
+    "model_params": "massW.dat",
+    "model_params_user": [
+    ],
+    "model_params_vars": {},
+    "fragment": ["Generator/ExternalLHEProducer.dat", "PartonShower/amcatnlo-pythia8.dat"],
+    "fragment_user": [],
+    "fragment_vars": {
+        "processParameters": [
+            "'TimeShower:nPartonsInBorn = 1'"
```

Comment:
add mmaxgamma 10?
@agrohsje how should we deal with this?

Reply:
Mmm isn't pythia's default mmaxgamma 10 already?

Reply:
as we are doing ttZ nlo with mll 1 for 22, we should have the maxgamma 1 gev cut for 22 and then increase to 4 afterwards. i don't know. :-(